site/src/sphinx/advanced-spring-webflux-integration.rst

```diff
@@ -0,0 +1,122 @@
+.. _advanced-spring-webflux-integration:
+
+Using Armeria with Spring WebFlux
+=================================
+
+.. note::
+
+    Visit `armeria-examples <https://github.com/line/armeria-examples>`_ to find the fully working example code.
+
+Spring framework provides powerful features which are necessary for building a web application, such as
+dependency injection, data binding, AOP, transaction, etc. In addition, if your Spring application integrates
+with Armeria, you can leverage the following:
+
+- Rich support for Apache `Thrift <https://thrift.apache.org/>`_ and `gRPC <https://grpc.io/>`_,
+  including the documentation service
+- Ability to run HTTP REST service and RPC service in the same port
+- Full HTTP/2 support for both server-side and client-side, including :ref:`the fancy web console <server-docservice>`
+  that enables you to send Thrift and gRPC requests from a web browser.
```

Comment:
- The current 'including ...' clause should be in the first bullet about Thrift and gRPC support.
- 'including `h2c` (plaintext HTTP/2)' ?

Reply:
Oops, my mistake. Thanks!